src/Payload/Level.php

```diff
@@ -2,6 +2,17 @@
 
 class Level implements \JsonSerializable
 {
+    const IGNORE = 'ignore';
```

Comment:
Aha! Thanks for the info. WOuld you mind adding a comment to this code and point to where they are mapped to the Rollbar levels?